core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1919,9 +1919,8 @@ abstract class RDD[T: ClassTag](
 
       val persistence = if (storageLevel != StorageLevel.NONE) storageLevel.description else ""
       val storageInfo = rdd.context.getRDDStorageInfo(_.id == rdd.id).map(info =>
-        "    CachedPartitions: %d; MemorySize: %s; ExternalBlockStoreSize: %s; DiskSize: %s".format(
-          info.numCachedPartitions, bytesToString(info.memSize),
-          bytesToString(info.externalBlockStoreSize), bytesToString(info.diskSize)))
+        "    CachedPartitions: %d; MemorySize: %s; DiskSize: %s".format(
+          info.numCachedPartitions, bytesToString(info.memSize), bytesToString(info.diskSize)))
```

Comment:
Since it was removed at `toString`, it's okay to remove it from `toDebugString`.
- https://github.com/apache/spark/pull/10752/files#diff-e49db126594e91593208902e13961dc09524442090801f4d6cfce7c43da13c72R42-R48